src/hooks/bindings/useDisableUpdater.ts

```diff
@@ -57,9 +57,12 @@ function useDisableUpdater(bindingUUID?: string) {
                             missingIndex ||
                             // Make sure there is a binding there at least
                             !spec.bindings[bindingIndex] ||
-                            // Try to be safe and make sure we have the right name
-                            getCollectionName(spec.bindings[bindingIndex]) !==
-                                collectionName
+                            // Try to be safe and make sure we have the right name when we are updating
+                            //  a specific binding
+                            (collectionName &&
+                                getCollectionName(
+                                    spec.bindings[bindingIndex]
+                                ) !== collectionName)
```

Comment:
Only need to check if the collection name is the same when the name is there to check :facepalm: